draft-decraeneginsberg-lsr-isis-fast-flooding.xml

```diff
@@ -363,32 +363,31 @@
 
             <t>LSPs</t>
 
-            <t>Complete Sequence Number PDUs (CSNPs) and Partial Sequence
-            Number PDUs (PSNPs)</t>
+            <t>Complete Sequence Number PDUs (CSNPs) and PSNPs</t>
           </list>Implementations today may prioritize the reception of Hellos
         over LSPs and SNPs in order to prevent a burst of LSP updates from
         triggering an adjacency timeout which in turn would require additional
         LSPs to be updated.</t>
 
-        <t>SNPs serve to acknowledge or trigger the transmission of specified
+        <t>CSNPs and PSNPs serve to acknowledge or trigger the transmission of specified
         LSPs. On a point-to-point link, PSNPs acknowledge the receipt of one
-        or more LSPs. Because PSNPs (like all IS-IS PDUs) use TLVs in the
+        or more LSPs. Because PSNPs use TLVs in the
         body, it is possible to acknowledge multiple LSPs using a single PSNP.
         For this reason, <xref target="ISO10589"/> specifies a delay
         (partialSNPInterval) before sending a PSNP so that the number of PSNPs
         required to be sent is reduced. On receipt of a PSNP, the set of LSPs
         acknowledged by that PSNP can be marked so that they do not need to be
         retransmitted.</t>
```

Comment:
The discussion of multiple acks in a PSNP seems redundant with
the text in Section 5.1.

Reply:
Removed one sentence: "Because PSNPs use TLVs in the body, it is possible to acknowledge multiple LSPs using a single PSNP."